specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10/examples/ReplicationJobs_Export.json

```diff
@@ -7,8 +7,8 @@
     "jobQueryParameter": {
       "startTime": "2017-04-27T14:26:51.9161395Z",
       "endTime": "2017-05-04T14:26:51.9161395Z",
-      "affectedObjectTypes": [],
```

Comment:
Do you know why this has changed types?

Reply:
The type changed from the initial version from array to string, so Model Validation was flagging it.
I can undo the non-formatting changes if you want though, since these fixes don't fully fix the model validation issues